code/api/src/modules/user/mutations.js

```diff
@@ -37,4 +37,37 @@ export const userRemove = {
     }
   },
   resolve: remove
-}
\ No newline at end of file
+}
+
+// User update
+export const updateUser = {
```

Comment:
Looks good! Really not that important, but to be consistent with all the other update functions, it would be 'userUpdate' instead of 'updateUser'.